<?xml version="1.0"?>
<launch>
    <include file="$(find assignment_2_2023)/launch/sim_w1.launch" />
    <param name="des_pos_x" value= "0.0" />
    <param name="des_pos_y" value= "5.0" />
    <param name="average_window" value= "20" />
    <node pkg="assignment_2_2023" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="assignment_2_2023" type="go_to_point_service.py" name="go_to_point"  />
    <node pkg="assignment_2_2023" type="bug_as.py" name="bug_action_service" output="screen" />
    <node pkg="assignment_2_2023" type="my_node_A.py" name="node_a" output="screen" />
    <node pkg="assignment_2_2023" type="my_node_B.py" name="node_b" output="screen" />
    <node pkg="assignment_2_2023" type="my_node_C.py" name="node_c" output="screen" />
    
</launch>
